README.md

```diff
@@ -64,4 +65,19 @@ tctest results 12345
 To wait for a running or queued build to complete and then show the results:
 ```bash
 tctest results 12345 --wait
+```
+
+*By Github PR Number*
+
+To show the PASS/FAIL/SKIP results for **all** TeamCity builds for a Github PR:
+```bash
+tctest pr-results 12345
```

Comment:
I think we could make this a sub command of pr?
```suggestion
tctest pr results 12345
```

Reply:
ohh yes make sense 👍